hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -185,10 +186,16 @@ public void notifyNotLeader(Collection<TransactionContext> pendingEntries) {
   public CompletableFuture<TermIndex> notifyInstallSnapshotFromLeader(
       RaftProtos.RoleInfoProto roleInfoProto, TermIndex firstTermIndexInLog) {
 
-    String leaderNodeId = RaftPeerId.valueOf(roleInfoProto.getFollowerInfo()
-        .getLeaderInfo().getId().getId()).toString();
+    String leaderNode = roleInfoProto.getFollowerInfo()
```

Comment:
In tests I see leaderNode coming ""
Received install snapshot notification from SCM leader:  with term index: (t:2, i:107)

Reply:
Addressed with latest ratis version bump